Cargo.toml

```diff
@@ -134,15 +134,15 @@ konst = { version = "0.3.8", default-features = false }
 lazy_static = "1.4.0"
 # TODO: upstreamed in https://github.com/ledger-community/rust-ledger/pull/9
 ledger-lib = { git = "https://github.com/heliaxdev/rust-ledger", rev = "f96f4559b3237d09218f7583df01acf36034ea79", default-features = false, features = ["transport_tcp"] }
-ledger-namada-rs = { git = "https://github.com/Zondax/ledger-namada", tag = "v0.0.24" }
+ledger-namada-rs = { git = "https://github.com/Zondax/ledger-namada", tag = "v1.0.5" }
 ledger-transport = "0.10.0"
 ledger-transport-hid = "0.10.0"
 libc = "0.2.97"
 libloading = "0.7.2"
 linkme = "0.3.24"
 # branch = "tomas/arbitrary"
-masp_primitives = { git = "https://github.com/anoma/masp", rev = "12ed8b060b295c06502a2ff8468e4a941cb7cca4" }
-masp_proofs = { git = "https://github.com/anoma/masp", rev = "12ed8b060b295c06502a2ff8468e4a941cb7cca4", default-features = false, features = ["local-prover"] }
+masp_primitives = { git = "https://github.com/anoma/masp", rev = "2914e6ff9a922bae8f1cb63a79d796a69af3d8aa" }
```

Comment:
could you make a branch from this rev? I'm seeing `This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.` for it which might cause issues in build

Reply:
Ah yes, I addressed your comment in the MASP crate and forgot to update the revision here. I've now merged `murisi/box-impl-build-params` into `main` in the MASP crate and changed the revision here to that. Thanks.